external/netty/README.md

```diff
@@ -0,0 +1,31 @@
+# External Netty Tests
+
+Netty tests are a part of the external third-party application tests that help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/[Issue #172](https://github.com/AdoptOpenJDK/openjdk-tests/issues/172) lists the applications that we have initially targeted to best exercise the Adoptium binaries. For each application, we choose to run a selection of their functional tests.
```

Comment:
One more request, please add an extra sentence that says:
Netty test material is pulled from the [netty](https://github.com/netty/netty.git) repository.

*please also hyperlink netty to the value from github_url in the test.properties file

Reply:
Done the following change, please review it.